src/tests/test_jsproxy.py

```diff
@@ -1155,3 +1155,13 @@ def test_memory_leaks(selenium):
         `);
         """
     )
+
+
+@run_in_pyodide
+def test_js_id():
+    from js import eval as run_js
+
+    [x, y, z] = run_js("let a = {}; let b = {}; [a, a, b]")
+    assert x.js_id == y.js_id
```

Comment:
BTW, we are currently not implementing `__eq__` for JsProxy, right?
Currently `x ==  y` evaluates to False as far as I can tell.   But that's a separate issue.

Reply:
Right, we should probably add equality checks in a separate PR.